src/flows/recommendation_api/corpus_candidate_sets/pockethits.py

```diff
@@ -21,13 +21,14 @@
 
 POCKETHITS_SQL = """
 SELECT
-    APPROVED_CORPUS_ITEM_EXTERNAL_ID as ID,
-    TOPIC
-FROM "SCHEDULED_CORPUS_ITEMS"
+    APPROVED_CORPUS_ITEM_EXTERNAL_ID as "ID",
+    TOPIC as "TOPIC"
+FROM "ANALYTICS"."DBT"."SCHEDULED_CORPUS_ITEMS"
```

Comment:
doesn't have to be here, but for publisher spread, eventually all these queries will need publisher fields